### DISCUSSION TITLE
想问一下如何卸载安装的依赖包……

### DISCUSSION THREAD

**Discussion by ghr0621** [Coolapk-UWP/Coolapk-UWP#163] [Q&A] [answered]
如题，大佬您好，我是一个纯小白，用您介绍的安装方法试图安装此应用，下载了那几个依赖包，但最终没有安装成功，现在想问一下如何删除那些依赖包。
谢谢大佬。

**Comment by wherewhere** [comment, marked as answer]
那些依赖都是UWP的常用依赖，如果擅自卸载会导致其他应用无法使用
